lrauv_system_tests/test/vehicle/test_sensor_timeinterpolation.cc

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
@arjo129 meta: why a separate source file, as opposed to adding a test case to `test_sensor.cc`?

Reply:
https://github.com/osrf/lrauv/issues/159 I cant load multiple test cases with the auv without :boom: happening.

Reply:
Ahh, that's right.